card/mod/search/set/abstract/wql_search.rb

```diff
@@ -43,23 +43,23 @@ def standardized_query_args args
   args
 end
 
-def wql_hash
-  @wql_hash = nil unless cache_query?
-  @wql_hash ||= wql_content.merge filter_and_sort_wql
+def cql_hash
```

Comment:
some of these are overridden and/or called in non-core mods, including wikirate mods. We'll have to fix those, too.

Reply:
Just saw that. I'll take a quick look Is this the only method name I changed, or are there more?